src/raftstore/store/fsm/store.rs

```diff
@@ -1060,15 +1058,16 @@ impl RaftBatchSystem {
             mailboxes.push((fsm.region_id(), BasicMailbox::new(tx, fsm)));
         }
         self.router.register_all(mailboxes);
+
         // Make sure Msg::Start is the first message each FSM received.
+        for addr in address {
```

Comment:
@5kbpers Is the adjustment OK? I recall that you have also changed the boot order of batch system.

Reply:
LGTM